### DISCUSSION TITLE
Apply css classes on specific page

### DISCUSSION THREAD

**Discussion by faizananwerali** [filamentphp/filament#8547] [Help]
I want to change tabs styling but only on specific pages. how do I do that?

```scss
.fi-tabs {
    @apply flex-wrap justify-center;
}
```

**Comment by leandrocfe** [comment]
Maybe you can add a custom class in the tab using `->extraAttributes(['class' => 'custom-tab'])`
then you can apply custom css

**Reply by faizananwerali** [reply]
I need to apply on the tabs class, not on a single tab.

**Reply by leandrocfe** [reply]
Did you try to add extraAttibutes on the tabs class?

**Reply by faizananwerali** [reply]
Yes, I did.

`Method 'extraAttributes' not found in \Filament\Resources\Pages\ListRecords\Tab` 

```php
ListRecords\Tab::make()->extraAttributes(['class' => 'custom-tab']])
```

Also, this is not what I want. I don't want to add classes to the individual tab, instead, I want to add to the parent div where the 'flex' class is added.

<img width="869" alt="Screenshot 2023-09-17 at 2 44 56 PM" src="https://github.com/filamentphp/filament/assets/12691366/db63c7e0-f3e4-4613-b385-452760710344">

If you look at the tabs' HTML. I want to target the parent div `fi-tabs`, not the `fi-tabs-items` class.

Currently, there is no way to add it from the livewire component.

If you look at the source code at `vendor/filament/support/resources/views/components/tabs/index.blade.php`.

```php
@props([
    'contained' => false,
    'label' => null,
])

<nav
    {{
        $attributes
            ->merge([
                'aria-label' => $label,
                'role' => 'tablist',
            ])
            ->class([
                'fi-tabs flex max-w-full gap-x-1 overflow-x-auto',
                'fi-contained border-b border-gray-200 px-3 py-2.5 dark:border-white/10' => $contained,
                'mx-auto rounded-xl bg-white p-2 shadow-sm ring-1 ring-gray-950/5 dark:bg-gray-900 dark:ring-white/10' => ! $contained,
            ])
    }}
>
    {{ $slot }}
</nav>
```

As you can clearly see there is no option to add custom classes.

That is why I'm looking to apply CSS classes for specific pages. So only those pages where I want CSS to be changed for 'Tabs' can be applied.

**Reply by leandrocfe** [reply]
Oh, sorry. I was thinking about [form tabs](https://filamentphp.com/docs/3.x/forms/layout/tabs)
